osf_tests/test_user.py

```diff
@@ -2140,7 +2132,7 @@ def test_can_gdpr_delete(self, user):
         assert user.is_disabled
         assert user.deleted is not None
 
-    def test_can_gdpr_delete_personal_nodes(self, user):
+    def test_can_gdpr_delete_personal_nodes(self, user, project):
```

Comment:
you don't seem to be using project!

Reply:
This is a pytest thing, just adding the param here creates the node, prior to this we weren't filtering out QFNs so, once we moved on from them I had to add a real node to fix that test.